db/migrate/20200729042607_create_discourse_voting_vote_counters.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class CreateDiscourseVotingVoteCounters < ActiveRecord::Migration[6.0]
+  def up
+    create_table :discourse_voting_vote_counters do |t|
+      t.integer :topic_id
+      t.integer :counter
```

Comment:
We can then rename this to `count`

Reply:
yeah, I wasn't sure about that, because someone can wrongly assume it is active record `count` method. Do you still vote for `count`? Just want to double-check :)

Reply:
Maybe `votes_count`? Btw am I right to say that this acts as a counter cache for `discourse_voting_votes`?

Reply:
lol I just realized if we name it `votes_count` it'll be `topic.vote_count.votes_count`.

Reply:
@lis2 Actually...... how about just adding the column to `topics` itself and call it `votes_count`. Then we can leverage on ActiveRecord's counter_cache feature.

Reply:
We also avoid having to join on an additional table.

Reply:
Interesting idea. I have a feeling that adding a column to core models by plugin may be a wrong idea. For sure code will benefit out of it but I think we should avoid it, if possible. 

My thinking is that if we have a new table with plugin name `prefix` if someone decides to uninstall votes after few years of use, we can reasonably easily clean (just drop tables).

Reply:
Good point. Maybe `discourse_voting_vote_counters.votes_count`?